emission/tests/exportTests/TestExportPipelineFull.py

```diff
@@ -0,0 +1,39 @@
+from future import standard_library
+standard_library.install_aliases()
+from builtins import *
+import os
+from os import path
+import emission.tests.common as etc
+import emission.exportdata.export_data as eeed
+import unittest
+import tempfile
+
+class TestExportPipelinePipelineFull(unittest.TestCase):
+    def testExportPipelineFull(self):
```

Comment:
you can just put these tests into the existing module. No need to have separate classes and files for them

Reply:
To clarify, should I move all tests into a common file? ie. tests/exporttests/TestExportModule.py

Reply:
Yup! in case you don't see the response!

Reply:
Single file pushed under new commit.